Modules/Core/Common/src/itkNumericTraits.cxx

```diff
@@ -15,6 +15,12 @@
  *  limitations under the License.
  *
  *=========================================================================*/
+
+#if !defined(ITK_LEGACY_REMOVE)
+// Supress MSVC warnings
```

Comment:
Shall I still do an amend/force-push to adjust the comment a little bit? It has a minor typo ("Supress"). No problem, of course.

Reply:
Yes, please fix the misspelling.